dashboard/app/helpers/levels_helper.rb

```diff
@@ -120,6 +120,9 @@ def app_options
 
     set_channel if @level.channel_backed?
 
+    # Always pass user age limit
+    view_options(is_13_plus: current_user && !current_user.under_13?)
```

Comment:
Hah. Good job Brent's brain on missing that, even though it was directly above this. :facepalm: